### DISCUSSION TITLE
Applying Data Retention Policies for ClickHouse in Homer-10

### DISCUSSION THREAD

**Discussion by jaydeep369** [sipcapture/homer#662] [General]
I followed the instructions from the [Homer Docker repository](https://github.com/sipcapture/homer-docker/tree/main/all-in-one) and successfully retrieved the required data on Grafana. All containers are up and running now. For the Heplify server, I’ve set HEPLIFYSERVER_DBDROPDAYS=5 to retain only 5 days of data.

However, I’m concerned about the data stored in the clickhouse-server data volume. Is there a way to apply a data retention policy in ClickHouse to ensure that only 5 days of data is retained there as well? 
Any guidance on configuring this would be greatly appreciated! Thanks..!

**Comment by lmangani** [comment]
Hello @jaydeep369 

> I’m concerned about the data stored in the clickhouse-server data volume.

[qryn](https://qryn.dev) is the backend project handling all the data and it has its own [settings](https://qryn.metrico.in/#/env) for controlling rotation and many other aspects. The default rotation is 7 days with default settings.

Hope this helps!
